File: app/src/main/java/com/madrasahdigital/walisantri/ppi67benda/model/notification/NotificationModel.java
package com.madrasahdigital.walisantri.ppi67benda.model.notification;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by Alhudaghifari on 11:16 22/05/19
 */
public class NotificationModel {
    @SerializedName("id")
    @Expose
    private Integer id;
    @SerializedName("subject")
    @Expose
    private String subject;
    @SerializedName("date")
    @Expose
    private String date;
    @SerializedName("read")
    @Expose
    private Boolean read;
    @SerializedName("url")
    @Expose
    private String url;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getSubject() {
        return subject;
    }

    public void setSubject(String subject) {
        this.subject = subject;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public Boolean getRead() {
        return read;
    }

    public void setRead(Boolean read) {
        this.read = read;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }
}

File: app/src/main/java/com/madrasahdigital/walisantri/ppi67benda/model/notification/NotificationModel.kt
package com.madrasahdigital.walisantri.ppi67benda.model.notification

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by Alhudaghifari on 11:16 22/05/19
 */
data class NotificationModel(
    @SerializedName("id")
    @Expose
    var id: Int = 0,

    @SerializedName("subject")
    @Expose
    var subject: String? = null,

    @SerializedName("date")
    @Expose
    var date: String? = null,

    @SerializedName("read")
    @Expose
    var isRead: Boolean = false,

    @SerializedName("url")
    @Expose
    var url: String? = null
)